modules/shared/src/main/scala/org/tessellation/schema/transaction.scala

```diff
@@ -41,6 +41,14 @@ object transaction {
   @newtype
   case class TransactionSalt(value: Long)
 
+  @derive(decoder, encoder, eqv, show)
+  case class TransactionData(
+    source: Address,
+    destination: Address,
+    amount: TransactionAmount,
+    fee: TransactionFee
+  )
+
   // TODO: figure out the Fiber usage for transaction
```

Comment:
If it's figured out I guess we can remove the TODO

Reply:
At the same time it's the definition, not usage here 🤔

Reply:
I think we can surely remove this TODO